package.json

```diff
@@ -85,7 +86,9 @@
     "react-cookie": "^4.1.1",
     "react-dom": "^18.2.0",
     "react-stately": "^3.23.0",
-    "uuid": "^9.0.0"
+    "request": "^2.88.2",
```

Comment:
Possibly related: the upstream bug that I think originally led us to add it is now also closed: https://github.com/googleapis/retry-request/issues/103#event-10621035485

Reply:
Thanks - I keep adding it back because building with various GCP-related modules complain about it (even though it's not listed as a dependency) - that PR makes me not want to take it! I'll make sure our deps are up to  date and see if it goes away, thanks.